data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -143,64 +160,81 @@ def _climate_factor(self) -> bool:
         # Low income: In 60th percentile or above for percent of block group population
         # of households where household income is less than or equal to twice the federal
         # poverty level. Source: Census's American Community Survey]
-        climate_criteria = (
-            (
-                self.df[
-                    field_names.EXPECTED_BUILDING_LOSS_RATE_FIELD
-                    + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-            | (
-                self.df[
-                    field_names.EXPECTED_AGRICULTURE_LOSS_RATE_FIELD
-                    + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-            | (
-                self.df[
-                    field_names.EXPECTED_POPULATION_LOSS_RATE_FIELD
-                    + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-        )
 
-        return (
+        criterion_columns = [
+            field_names.FEMA_LOSS_RATE_LOW_INCOME, 
+            field_names.AGRICULTURE_LOSS_RATE_LOW_INCOME,
+            field_names.EXPECTED_BUILDING_LOSS_RATE_LOW_INCOME
+        ]
+
+        self.df[field_names.FEMA_LOSS_RATE_LOW_INCOME] = (
             self.df[
-                field_names.POVERTY_LESS_THAN_200_FPL_FIELD
+                field_names.EXPECTED_POPULATION_LOSS_RATE_FIELD
                 + field_names.PERCENTILE_FIELD_SUFFIX
             ]
-            >= self.LOW_INCOME_THRESHOLD
-        ) & climate_criteria
+            >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
+        ) & self.FPL_200_SERIES
+
+        self.df[field_names.AGRICULTURE_LOSS_RATE_LOW_INCOME] = (
+            self.df[
+                field_names.EXPECTED_AGRICULTURE_LOSS_RATE_FIELD
+                + field_names.PERCENTILE_FIELD_SUFFIX
+            ]
+            >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
+        ) & self.FPL_200_SERIES
+
+        
+        self.df[field_names.EXPECTED_BUILDING_LOSS_RATE_LOW_INCOME] = (
+            self.df[
+                field_names.EXPECTED_BUILDING_LOSS_RATE_FIELD
+                + field_names.PERCENTILE_FIELD_SUFFIX
+            ]
+            >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
+        ) & self.FPL_200_SERIES
+
+
+        self.df[field_names.THRESHOLD_COUNT] += self.df[criterion_columns].sum(axis = 1)
+
+        # this is one idea for a pattern I was thinking of using
+        return self.df[criterion_columns].any(axis='columns')
```

Comment:
This looks awesome to me! I love all of this. To me this climate factor now looks like a pattern we can follow on all the factors!

Reply:
Sweet - done.